src/main/webapp/app/shared/monaco-editor/monaco-editor.component.ts

```diff
@@ -44,13 +48,35 @@ export class MonacoEditorComponent implements OnInit, OnDestroy {
             minimap: { enabled: false },
             readOnly: this._readOnly,
             lineNumbersMinChars: 4,
+            scrollBeyondLastLine: false,
+            scrollbar: {
+                alwaysConsumeMouseWheel: false, // Prevents the editor from consuming the mouse wheel event, allowing the parent element to scroll.
+            },
         });
+        this._editor.getModel()?.setEOL(monaco.editor.EndOfLineSequence.LF);
         renderer.appendChild(elementRef.nativeElement, this.monacoEditorContainerElement);
     }
 
     @Input()
     textChangedEmitDelay?: number;
 
+    // TODO: This is only necessary due to a bug in the code editor layout. In the future, we can remove this and the corresponding CSS class.
```

Comment:
wasn't this the bug of the monaco editor itself? If you maybe you can link your issue here which you created in the monaco github repository. This way everybody knows what you are referring to.

Reply:
I'll update the comment to be more precise. Solutions involving grid layouts with the Monaco editor (like vscode.dev) typically handle the width/height of differently than we do, so they don't encounter this problem (-> there is also no issue to link). I think we need to change our code editor grid a bit to resolve this. Luckily, this problem did not affect the markdown editor.